benchopt/runner.py

```diff
@@ -212,16 +212,17 @@ def run_one_solver(benchmark, dataset, objective, solver, n_repetitions,
         # Get meta
         meta = dict(
             objective_name=str(objective),
+            **objective._parameters,
```

Comment:
The issue with this approach is that you have a risk of overwritting the parameters if you have two that have the same name in `dataset/solver/objective`.
As these are user defined, I'd rather add `obj_/dataset_/solver_` prefix to the keys